File: platforms/web/examples/core/pass/add_mesh.js
import { Pass, Shader, Mesh, Vertex } from "fragmentcolor";

const mesh = new Mesh();
mesh.addVertex(Vertex.new([0.0, 0.0]));

const shader = Shader.fromMesh(mesh);
const pass = new Pass("pass"); pass.addShader(shader);

pass.addMesh(mesh).expect("mesh is compatible");

File: platforms/python/examples/core/pass/add_mesh.py
from fragmentcolor import Pass, Shader, Mesh, Vertex

mesh = Mesh()
mesh.add_vertex(Vertex([0.0, 0.0]))

shader = Shader.from_mesh(mesh)
rpass = Pass("rpass"); rpass.add_shader(shader)

rpass.add_mesh(mesh).expect("mesh is compatible")